### ISSUE TITLE
Graviton/ARM machine support in Tableau Hyper API

### ISSUE THREAD

**Issue by theiconik** [tableau/hyper-db#80]
Hi team, is there any plan for release of Tableau Hyper API for Graviton/ARM machine support?
I checked the following issue : [https://github.com/tableau/hyper-api-samples/issues/100](https://github.com/tableau/hyper-api-samples/issues/100), seems like there was no plan last year.

If there's no plan, is there any workaround to run Hyper process on Graviton/ARM machines?

**Comment by alendit** [comment]
Hi, thanks for reaching out. Unfortunately, Graviton is currently not on our roadmap. 

We'll keep this issue open to track the progress, should we ever get to it.

**Comment by Tonkonozhenko** [comment]
@alendit what about apple m1+ laptops? it's impossible to run the native library on the last 3 generations of Apple laptops

**Comment by alendit** [comment]
Hi @Tonkonozhenko, Apple Silicon support is high on our backlog and we hope to get back to you with good news soon.

**Comment by Tonkonozhenko** [comment]
Hi @alendit, thanks for the good news, we'll be waiting

**Comment by alendit** [comment]
Hi @Tonkonozhenko, Apple Silicon support was add in [the latest release](https://tableau.github.io/hyper-db/docs/releases#0019484-june-6-2024). You can download it [here](https://downloads.tableau.com/tssoftware//tableauhyperapi-0.0.19484-py3-none-macosx_13_0_arm64.whl).

**Comment by jwhitaker-gridcog** [comment]
Awesome to hear about Mac support! Here's another vote for arm though, tableauhyperapi is literally the thing blocking us from deploying on graviton.

**Comment by lorepirri** [comment]
Hi @alendit is Linux/aarch64 on the roadmap?

**Comment by neontty** [comment]
agreed, linux/arm64 is critical. Can we help creating that artifact?

**Comment by diwu-sf** [comment]
Docker on Mac still uses linux/arm64  underneath so we need linux/arm64  support.
